### PR TITLE
Saving profiles to json file

### PR DESCRIPTION
**Pull request by SekayZ**
Added the user_profiles file in src/main/resources where the users are saved in JSON format.

### CODE REVIEW

**Comment by willyyhuang** [comment]
get latest master

File: SmartHomeSimulatorBack/SmartHomeSimulator/src/main/java/com/project/SmartHomeSimulator/module/SimulationContext.java [hunk 1]
```diff
@@ -97,8 +102,22 @@ public void setCurrentSimulationUser(User currentSimulationUser) {
 	}
 
 	public List<User> getSimulationUsers() {
+		counter++;
+		if (counter == 1) {
+			loadUserProfiles();
```
Comment (Karin-kazarian) [inline SmartHomeSimulatorBack/SmartHomeSimulator/src/main/java/com/project/SmartHomeSimulator/module/SimulationContext.java:107]: I think it's better if you put this method in the constructor instead. That way the users will be loaded once the simulation context is created. And you won't need the counter anymore.

File: SmartHomeSimulatorBack/SmartHomeSimulator/src/main/java/com/project/SmartHomeSimulator/model/User.java [hunk 2]
```diff
@@ -1,40 +1,63 @@
 package com.project.SmartHomeSimulator.model;
 
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
 import org.springframework.boot.jackson.JsonComponent;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
```
Comment (Karin-kazarian) [inline SmartHomeSimulatorBack/SmartHomeSimulator/src/main/java/com/project/SmartHomeSimulator/model/User.java:9]: make sure to delete the unused imports

**Review by Karin-kazarian — CHANGES_REQUESTED** [review]

**Review by Karin-kazarian — APPROVED** [review]

**Review by jenniferasmar — APPROVED** [review]
works! good job

**Pull request merged**